CMakeLists.txt

```diff
@@ -13,21 +18,22 @@ set(main_exe_name RigelEngine)
 # Project settings
 ###############################################################################
 
-option(USE_GL_ES "Use OpenGL ES instead of regular OpenGL" OFF)
+option(USE_GL_ES "Use OpenGL ES instead of regular OpenGL" ON)
 option(WARNINGS_AS_ERRORS "Treat compiler warnings as errors" ON)
 
-
 # Dependencies
 ###############################################################################
 
-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
+# set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "/home/pratikone/git/emsdk/upstream/emscripten/cmake/Modules/")
 
-set(Boost_USE_STATIC_LIBS ON)
-find_package(Boost 1.65 COMPONENTS program_options REQUIRED)
-find_package(SDL2 REQUIRED)
-find_package(SDL2_mixer REQUIRED)
+# set(Boost_USE_STATIC_LIBS ON)
```

Comment:
All these `find_package` calls should go into an `if("${CMAKE_SYSTEM_NAME}" STREQUAL "Emscripten")`

Reply:
https://emscripten.org/docs/compiling/Building-Projects.html#detecting-emscripten-in-preprocessor

Reply:
> there is an equivalent variable for Cmake too

Yes, I'm using it in the code snippet in my comment above (checking `CMAKE_SYSTEM_NAME`). Or are you referring to a different variable?

Reply:
no the same one